libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-data-manager-adapter.directive.ts

```diff
@@ -54,7 +56,9 @@ export class SkyAgGridDataManagerAdapterDirective
 
   constructor(
     private changeDetector: ChangeDetectorRef,
-    private dataManagerSvc: SkyDataManagerService
+    private dataManagerSvc: SkyDataManagerService,
+    private elementRef: ElementRef,
+    private topHorizontalScrollService: SkyAgGridTopScrollService
```

Comment:
Can we change this to `topScrollSvc` to match the removal of "horizontal" elsewhere and to keep the same abbreviation as `dataManagerSvc` above?

Reply:
Updated.